src/Sarif.Multitool/Rules/SARIF2009.ConsiderConventionalIdentifierValues.cs

```diff
@@ -34,23 +34,13 @@ public class ConsiderConventionalIdentifierValues : SarifValidationSkimmerBase
         public override MultiformatMessageString FullDescription => new MultiformatMessageString { Text = RuleResources.SARIF2009_ConsiderConventionalIdentifierValues_FullDescription_Text };
 
         protected override IEnumerable<string> MessageResourceNames => new string[] {
-            nameof(RuleResources.SARIF2009_ConsiderConventionalIdentifierValues_Note_UseConventionalRuleIds_Text),
-            nameof(RuleResources.SARIF2009_ConsiderConventionalIdentifierValues_Note_UseConventionalUriBaseIdNames_Text)
+            nameof(RuleResources.SARIF2009_ConsiderConventionalIdentifierValues_Note_UseConventionalRuleIds_Text)
         };
 
         public override FailureLevel DefaultLevel => FailureLevel.Note;
 
-        private static readonly string[] s_conventionalSymbols = new string[] { "REPOROOT", "SRCROOT", "TESTROOT", "BINROOT" };
         private static readonly Regex s_conventionalIdRegex = new Regex(@"^[A-Z]{1,5}[0-9]{1,4}$", RegexOptions.Compiled | RegexOptions.CultureInvariant);
 
-        protected override void Analyze(Run run, string runPointer)
-        {
-            if (run.OriginalUriBaseIds != null)
-            {
-                AnalyzeOriginalUriBaseIds(run.OriginalUriBaseIds, runPointer.AtProperty(SarifPropertyName.OriginalUriBaseIds));
-            }
-        }
-
         protected override void Analyze(Tool tool, string toolPointer)
```

Comment:
>Analyze [](start = 32, length = 7)

While we're in here, let's back this one up to the `run` level like we did for -- whatever that other rule was earlier today. We are only about `run.tool.driver.rules`.